.github/workflows/edisonCI.yml

```diff
@@ -0,0 +1,40 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python package
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
```

Comment:
- should this be pip3 or are you sure that there is only python 3 installed?
- why are you sometimes using python -m pip and sometimes pip directly?

Reply:
This is probably coming from the default GitHub template

Reply:
yes. this is the Github Template, Do you want me to change it either way?

Reply:
I would like the edison team to come up with a standard of how to do things.
You should change the github template if it is not following that standard.

In setup.sh and in the unmerged file install_flask.sh, pip is install via the package manager. (sudo apt-get install -y python3-pip). That means that you can't use pip to upgrade itself, or it will cause problems.
In this file, you're installing flask as part of a requirements.txt file. In the setup.py and the unmerged install_flask.sh and install_postgresql.sh  files it installs them without a requirements.txt file.

Reply:
we decided that we use in requirements file